tests/integration/connect-test.js

```diff
@@ -88,47 +113,35 @@ test('each computed is truly readonly', function(assert) {
     assert.throws(() => {
       this.$('.btn-alter').trigger('click');
     }, (e) => {
-      return e.message.indexOf('Cannot set read-only property') > -1;
+      return e.message.indexOf('Cannot set redux property "low"') > -1;
     });
   });
 });
 
 test('lifecycle hooks are still invoked', function(assert) {
```

Comment:
This test even possible at the integration level (without adding additional monkey patching/hacks to confirm the lifecycle hooks are indeed invoked?) Example: if we had kept the willDestory/ didUpdateAttrs separate we would be able to see this more easily correct? One of the reasons I prefer those explicit hooks is that they allow me (the test driven developer) an entry point to do something (here for verification/ but you can imagine other scenarios I'm sure)